all-repos.yaml

```diff
@@ -198,3 +198,4 @@
 - https://github.com/janosh/format-ipy-cells
 - https://github.com/FalcoSuessgott/lint-gitlab-ci
 - https://github.com/comkieffer/xml-linter-hook
+- https://github.com/klieret/jekyll-relative-url-hook/
```

Comment:
- none of the other urls have a trailing slash
- hook ids should be `dashed-names`
- entrypoints should be `dashed-names`
-